src/dsp_tools/commands/xmlupload/stash/upload_stashed_resptr_props.py

```diff
@@ -37,7 +37,7 @@ def upload_stashed_resptr_props(
     print(f"{datetime.now()}: Upload the stashed resptrs...")
     logger.info("Upload the stashed resptrs...")
     not_uploaded: list[LinkValueStashItem] = []
-    for res_id, stash_items in stashed_resptr_props.res_2_stash_items.items():
+    for res_id, stash_items in stashed_resptr_props.res_2_stash_items.copy().items():
```

Comment:
**question**: 
Would it not also be necessary to add this exception handling here if the upload were interrupted here?